src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -76,21 +82,6 @@ var HTMLDOMPropertyConfig = {
     httpEquiv: 0,
     // Attributes with mutation methods must be specified in the whitelist
     value: 0,
-    // The following attributes expect boolean values. They must be in
-    // the whitelist to allow boolean attribute assignment:
-    autoComplete: 0,
```

Comment:
Removed autoComplete. It is actually `on/off`, not `true/false`. Thanks @syranide!